### PR TITLE
Add black list

### PR DESCRIPTION
**Pull request by yunfangjuan**
- get black list from the vector.gz file 
- change the genus_tax_id to -201 for all the black listed species ids.

### CODE REVIEW

File: config/taxon_blacklist.txt [hunk 1]
```diff
@@ -0,0 +1,597 @@
+-1
```
Comment (boris-dimitrov) [inline config/taxon_blacklist.txt]: is -1 always blacklisted?  what if that's something genuinely never seen before? :P)

Comment (boris-dimitrov) [reply config/taxon_blacklist.txt]: i think right now -1 goes under genus -200 rather than -201 but anyway we will need to reaggregate

Comment (boris-dimitrov) [reply config/taxon_blacklist.txt]:
```
mysql> select * from taxon_lineages where taxid < 0;
Empty set (0.00 sec)
```
So this means -1 will always be put under -200, might as well exclude it from -201.

**Comment by yunfangjuan** [comment]
can you disable the submit button once i clicked on it. It should prolly say submitting when doing it.

**Pull request merged**